packages/web-components/project.json

```diff
@@ -3,5 +3,8 @@
   "$schema": "../../node_modules/nx/schemas/project-schema.json",
   "projectType": "library",
   "implicitDependencies": [],
-  "tags": ["platform:web", "web-components"]
+  "tags": ["platform:web", "web-components"],
+  "targets": {
+    "e2e": { "dependsOn": ["build-storybook"] }
```

Comment:
wc depends on having storybook build first ( also doesn't use browser component tests as rest of monorepo )